deploy-cluster-aws/startup.sh

```diff
@@ -1,28 +1,33 @@
 #! /bin/bash
 
-# The set -e option instructs bash to immediately exit if any command has a non-zero exit status
+# The set -e option instructs bash to immediately exit
+# if any command has a non-zero exit status
 set -e
 
 function printErr()
     {
-        echo "usage: ./startup.sh <tag> <number_of_nodes_in_cluster>"
+        echo "usage: ./startup.sh <tag> <number_of_nodes_in_cluster> <pypi_or_branch>"
         echo "No argument $1 supplied"
     }
 
-if [ -z "$1" ]
-  then
+if [ -z "$1" ]; then
     printErr "<tag>"
     exit 1
 fi
 
-if [ -z "$2" ]
-  then
+if [ -z "$2" ]; then
     printErr "<number_of_nodes_in_cluster>"
     exit 1
 fi
 
+if [ -z "$3" ]; then
```

Comment:
I would keep `$3` optional, with `pypi` as default value (this should be the most common case).